### PR TITLE
Branch4

### PR DESCRIPTION
**Pull request by zarembaolesia**
Changed tests to use DataProvider and moved methods to Person class as it seemed more logical.

### CODE REVIEW

File: src/test/java/PeopleTest.java [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+            Assert.assertNotEquals(person1.getLastName(), person2.getLastName(), "Last names are equal");
+            Assert.assertEquals(person2.getLastName(), person2.getPrevLastName(), "Men`s last name changed when woman divorced him");
+        } else if (person1 instanceof Man && person2 instanceof Woman) {
+            Assert.assertEquals(person2.getLastName(), person2.getPrevLastName(), "Woman didn`t change back her last name");
+            Assert.assertNotEquals(person2.getLastName(), person1.getLastName(), "Last names are equal");
+            Assert.assertEquals(person1.getLastName(), person1.getPrevLastName(), "Men`s last name changed when he divorced a woman");
+        }
+    }
+
+//    @Test
```
Comment (yurii-voronenko) [inline src/test/java/PeopleTest.java:55]: Remove comments. Usually you do not neet to commit comments

File: src/test/java/PeopleTest.java [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+        if (person1 instanceof Woman && person2 instanceof Man) {
+            Assert.assertEquals(person1.getLastName(), person1.getPrevLastName(), "Woman didn`t change back her last name");
+            Assert.assertNotEquals(person1.getLastName(), person2.getLastName(), "Last names are equal");
+            Assert.assertEquals(person2.getLastName(), person2.getPrevLastName(), "Men`s last name changed when woman divorced him");
+        } else if (person1 instanceof Man && person2 instanceof Woman) {
+            Assert.assertEquals(person2.getLastName(), person2.getPrevLastName(), "Woman didn`t change back her last name");
+            Assert.assertNotEquals(person2.getLastName(), person1.getLastName(), "Last names are equal");
+            Assert.assertEquals(person1.getLastName(), person1.getPrevLastName(), "Men`s last name changed when he divorced a woman");
+        }
+    }
```
Comment (yurii-voronenko) [inline src/test/java/PeopleTest.java:53]: Test ofr retirement is absent, add test of boundary values for man and woman

**Review by yurii-voronenko — APPROVED** [review]
Done
